src/schemathesis/specs/openapi/negative.py

```diff
@@ -0,0 +1,65 @@
+from copy import deepcopy
+from typing import Any, Callable, Dict, TypeVar
+
+from hypothesis import strategies as st
+from hypothesis.strategies._internal.featureflags import FeatureStrategy
+from hypothesis_jsonschema import from_schema
+
+T = TypeVar("T")
+Draw = Callable[[st.SearchStrategy[T]], T]
+Schema = Dict[str, Any]
+# draw, new_schema, property schema, name
+Mutation = Callable[[Draw, Schema, Schema, str], None]
+
+
+def negative_schema(schema: Dict[str, Any]) -> st.SearchStrategy:
+    """A strategy for objects, that DO NOT match the input schema."""
+
+    # First we mutate the input schema, so any valid instance for it is invalid to the original schema
+
+    @st.composite  # type: ignore
+    def inner(draw: Draw) -> Any:
+        new_schema = deepcopy(schema)
+        properties = sorted(new_schema.get("properties", {}).items())
+        # At least one mutation should be done to at least one property
+        mutated_property_name, mutated_property_schema = draw(st.sampled_from(properties))
+        mutation = draw(st.sampled_from(MUTATIONS))
+        mutation(draw, new_schema, mutated_property_schema, mutated_property_name)
+
+        # Other properties & mutations are chosen with feature flags
+        enabled_properties = draw(FeatureStrategy())
+        enabled_mutations = draw(FeatureStrategy())
```

Comment:
So long as your names don't collide, you can use a single `FeatureStrategy` for both.

Unclear if you'd want to, but you could also use `st.shared(FeatureStrategy(), "key")` to have consistent feature exclusions across everything in the test case.

Reply:
> So long as your names don't collide, you can use a single FeatureStrategy for both.

Do you mean checking it at runtime? I.e., if there are collisions, then create two strategies, otherwise create one for both?

> Unclear if you'd want to, but you could also use st.shared(FeatureStrategy(), "key") to have consistent feature exclusions across everything in the test case.

Makes sense! I probably missed the scope of Swarm testing initially - as far as I understand now, the feature exclusion should happen on the test case level, so some features remain disabled during the whole test case, and this basically what makes the Swarm testing work so well :)

Reply:
I mean "just create one, and it'll behave oddly-but-acceptably if a property is named `mutation_type` or whatever.

> Makes sense! I probably missed the scope of Swarm testing initially - as far as I understand now, the feature exclusion should happen on the test case level, so some features remain disabled during the whole test case, and this basically what makes the Swarm testing work so well :)

I don't know of experiments about swarm testing at various levels - heirarchal swarm testing seems intuitively reasonable (though would perform poorly with Hypothesis' current implementation), and global-per-test-case seems like the best option.